src/js/fs.js

```diff
@@ -857,9 +857,16 @@ WriteStream.prototype._write = function(data, encoding, cb) {
       if (this.autoClose) {
         this._destroy();
       }
-      return cb(er);
+      process.nextTick(onwrite);
+      if (util.isFunction(cb)) {
+        process.nextTick(function() {
+          cb(er);
+        });
+      }
```

Comment:
This did copy from here: https://github.com/yodaos-project/ShadowNode/blob/master/src/js/net.js#L172-L177